nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/pom.xml

```diff
@@ -120,4 +124,99 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
+
+
+    <profiles>
```

Comment:
I wanted to keep it contained for right now to evaluate. I can move this up to the root.